config/detekt/baseline.yml

```diff
@@ -2390,6 +2392,7 @@
     <ID>MagicNumber:BackupLogic.kt$BackupLogic$0.7</ID>
     <ID>MagicNumber:BackupLogic.kt$BackupLogic$0.8</ID>
     <ID>MagicNumber:BackupLogic.kt$BackupLogic$0.9</ID>
+    <ID>MagicNumber:BackupWorker.kt$BackupWorker$5</ID>
```

Comment:
This is valid Detekt feedback. Can you revert the baseline and fix as many Detekt errors as possible? The ones in your module seems valid.